### ISSUE TITLE
Microphone question, Safari request

### ISSUE THREAD

**Issue by oakniiv** [kyleneideck/BackgroundMusic#557]
> There's no template for this issue type. I just wanted to make it clear that it's OK to submit other types of issues.
Does it actually access my physical microphone or just virtually the channel? The yellow light alerting me the whole time that my microphone is recording kind of scary.

Support for Safari? Idk how the app works in the background but if you can not pause the audio inside safari, would it be possible to just mute safari as the whole?

Thank you so much for this, btw. works on the M1 perfectly.

**Comment by miraclx** [comment]
Yeah, I have the same concern. The yellow light always on isn't great.
